packages/3dt/buildinfo.json

```diff
@@ -1,13 +1,15 @@
 {
-  "requires": ["exhibitor"],
-  "docker": "golang:1.7",
-  "single_source" : {
-    "kind": "git",
-    "git": "https://github.com/dcos/3dt.git",
-    "ref": "c9a7755fa75bd1c96192a26daa6a6fea0c714001",
+  "docker": "golang:1.7", 
+  "group": "systemd-journal", 
```

Comment:
Can we get an integration test the journal fetching is actually working as well as one that 3dt can actually get the mesos ID of an agent as part of the integration test suite in this repo?

Want to prevent things like forgetting this journal line from happening again.